server/__tests__/products_test.js

```diff
@@ -5,20 +5,30 @@ describe('GET /api/products', () => {
 	test('responds with json and 200 status', function (done) {
 		request(app)
 			.get('/api/products/')
-			// .expect('Content-Type', /json/)
+			.expect('Content-Type', /json/)
 			.expect(200, done);
 	});
 })
+
 describe('PUT /api/products', () => {
-	test('accepts id and shop_quantity and responds with 200 status', function (done) {
+	test('responds with 200 status', function (done) {
 		request(app)
 			.put('/api/products/1')
 			.send({
 				id: 1,
-				shop_quantity: 11,
+				shop_quantity: 9,
 			})
 			.set('Accept', 'application/json')
-			.expect(200)
-		done();
+			.expect(200, done);
+	});
+})
+
+describe('PUT /api/products', () => {
+	test('responds with 404 status', function (done) {
```

Comment:
See comment above. Test description could be more descriptive.

Reply:
Ah yes, it isn't descriptive at all. I'll change this, thanks